src/mongoose_commands.erl

```diff
@@ -676,6 +683,8 @@ check_caller(Caller, #mongoose_command{caller_pos = CallerPos}, Args) ->
         ACal ->
             ok;
         _ ->
-            throw(caller_jid_mismatch)
+            throw({denied, "Caller ids do not match"})
     end.
 
+sizeof(#{} = M) -> maps:size(M);
+sizeof([_|_] = L) -> length(L).
```

Comment:
Your editor is not adding end-of-lines, this is typical of MS, like with VSC. Unix traditionally does and git despises these end-of-lines missing 😄